api/v1alpha1/backendtrafficpolicy_types.go

```diff
@@ -100,16 +100,15 @@ type BackendTrafficPolicySpec struct {
 	// +notImplementedHide
 	// +optional
 	RequestBuffer *RequestBuffer `json:"requestBuffer,omitempty"`
-	// Telemetry configures the telemetry settings for the backend or backend.
+	// Telemetry configures the telemetry settings for the backend or HTTPRoute.
```

Comment:
Can we set tracing for an individual backend?

Reply:
suggest by https://github.com/envoyproxy/gateway/pull/5672#discussion_r2036156305

Reply:
I didn't see any decisions in that comment.

Reply:
that suggestion was around removing backend and use route instead. updating with a suggestion